
export function createException(name, message) {
  const msg = message || '';
  const err = new Error(msg);
  err.name = name;
  return err;
}

export function prependException(e, prefix) {
  const oldMsg = e.message;
  e.message = `${prefix}\n${oldMsg}`;
  return e;
}

